backend/pkg/modules/internal/bfla/bfladetector/learn_and_detect_bfla_test.go

```diff
@@ -115,9 +115,9 @@ func initBFLADetector(ctrl *gomock.Controller, backendAccessor *core.MockBackend
 	}).AnyTimes()
 	backendAccessor.EXPECT().CreateAPIEventAnnotations(ctx, gomock.Any(), gomock.Any(), gomock.Any()).AnyTimes()
 	backendAccessor.EXPECT().GetAPIInfoAnnotation(ctx, gomock.Any(), gomock.Any(), gomock.Any()).AnyTimes()
-	ctrlNotifier := bfladetector.NewControllerNotifier(backendAccessor)
+	ctrlNotifier := bfladetector.NewControllerNotifier("bfla", backendAccessor)
```

Comment:
Several questions:
1. why the method called ControllerNotifier?
2. maybe it's better to have the names returned statically inside the modules in the Name() method?
3. should these names align to the generated API enums?

Reply:
sounds good :)